esp32c3/src/assist_debug/core_0_debug_mode.rs

```diff
@@ -28,14 +28,14 @@ impl core::fmt::Debug for R {
             .finish()
     }
 }
-#[doc = "ASSIST_DEBUG_C0RE_0_DEBUG_MODE\n\nYou can [`read`](crate::Reg::read) this register and get [`c0re_0_debug_mode::R`](R). See [API](https://docs.rs/svd2rust/#read--modify--write-api)."]
-pub struct C0RE_0_DEBUG_MODE_SPEC;
-impl crate::RegisterSpec for C0RE_0_DEBUG_MODE_SPEC {
+#[doc = "ASSIST_DEBUG_C0RE_0_DEBUG_MODE\n\nYou can [`read`](crate::Reg::read) this register and get [`core_0_debug_mode::R`](R). See [API](https://docs.rs/svd2rust/#read--modify--write-api)."]
```

Comment:
```suggestion
#[doc = "ASSIST_DEBUG_CORE_0_DEBUG_MODE\n\nYou can [`read`](crate::Reg::read) this register and get [`core_0_debug_mode::R`](R). See [API](https://docs.rs/svd2rust/#read--modify--write-api)."]
```

I'm not sure where these docs come from but some of these still contain a 0.

Reply:
Another one :)

Reply:
There is actually a lot more 😅